### DISCUSSION TITLE
Is there anyway to batch the inpaint process?

### DISCUSSION THREAD

**Discussion by kmasterCross** [Bing-su/adetailer#196] [Q&A]
Currently, even if I generate 1 batch of 8 images, this extension will inpaint them 1 by 1. So it ends up taking much longer than 2X the time. Is there a setting where I can align the batch size as well?

**Comment by Bing-su** [comment]
Right now it seems like it's not possible. I'll look into it further.

**Reply by Himmelstiger** [reply]
Not sure if it's relevant, but it's possible in Automatic1111 Version 1.6 and 1.7. Just go to img2img, than look at Generation-tab (beyond prompt inputs) and than look for the "Batch"-tab. There you find the function you search for.

**Comment by ecs-molty** [comment]
## Status update — 2026-04-03

### Previously created issues (all closed)
- #5313 Mailbox utilization sync ✅
- #5314 Defender vulnerability metrics ✅
- #5315 Defender incidents/alerts ✅
- #5316 Last business review field ✅
- #5317 Halo SLA + CSAT ✅
- #5318 N-Able monitor status rollup ✅

### In progress
- #5312 Per-client FortiGate integration — spec updated today (encrypted credentials in DDB, link-only on org, sync-now endpoint, fresh fetch on page visit). Ready to implement.
- #5346 Locations page FortiGate linking — unblocked, depends on #5312
- #5311 Per-location FortiGate health monitoring — still open question on Lambda→FortiGate network reachability

### New issues created today (missing from spec)
- #5385 Client Health Report page (`/dashboard/organizations/{id}/health`) — the main report view that ties everything together
- #5386 Known risks from Hudu custom assets
- #5387 License over/under-provisioning check
- #5388 Device cross-system presence check (N-Able vs Halo)
- #5389 Location/site cross-reference (Halo sites vs N-Able sites)
- #5390 Last proactive contact field (manual AM entry)

### Not yet issued (by design)
- EHR (Effective Hourly Rate) — being built as a separate report; will surface result in #5385 once available
- Last field visit — computable from Halo team_id=19; will be derived in #5385 directly (no separate sync needed)

All items above are in the game loop queue. The report page (#5385) should be built last — after its data sources are ready.
